apps/condo/domains/user/schema/AuthenticateUserWithPhoneAndPasswordService.js

```diff
@@ -13,8 +14,34 @@ const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
 
 const redisGuard = new RedisGuard()
 
-const GUARD_WINDOW_SIZE_SEC = 60 * 60 // seconds
-const GUARD_WINDOW_LIMIT = 10
+const GUARD_DEFAULT_WINDOW_SIZE_SEC = 60 * 60 // seconds
```

Comment:
We usually add `in` part before `sec / ms` for such constants name:
 
```suggestion
const GUARD_DEFAULT_WINDOW_SIZE_IN_SEC = 60 * 60 // seconds
```